3 - PB/Documentazione esterna/Piano di Qualifica/integrationTest.json

```diff
@@ -0,0 +1,152 @@
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-5.5",
+      "Descrizione":"productRepository: verifica che venga correttamente restituito un array vuoto quando non viene trovato nessun prodotto",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-5.6",
+      "Descrizione":"productRepository: verifica che venga correttamente restituito null quando non vengono trovati prodotti cercando per ID",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-6.1",
+      "Descrizione":"zoneRepository: verifica che vengano correttamente restituite tutte le zone",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-6.2",
+      "Descrizione":"zoneRepository: verifica che vengano correttamente restituite tutte le zone che non contengono prodotti",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-6.3",
+      "Descrizione":"zoneRepository: verifica che venga correttamente restituita la zona cercata per ID",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-6.4",
+      "Descrizione":"zoneRepository: verifica che venga correttamente gestito un errore quando si vogliono restituire tutte le zone",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-6.5",
+      "Descrizione":"zoneRepository: verifica che venga correttamente gestito un errore quando si vuole restituire una zona cercata per ID",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-6.6",
+      "Descrizione":"zoneRepository: verifica che venga correttamente restituito un array vuoto quando non vengono trovate zone",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-6.7",
+      "Descrizione":"zoneRepository: verifica che venga correttamente restituito null quando non viene trovata la zona cercata per ID",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-7.1",
+      "Descrizione":"readSavedSVG: verifica che venga correttamente letto il contenuto del file SVG",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-8.1",
+      "Descrizione":"readSavedSVG: verifica che venga correttamente salvato il contenuto del file SVG",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-9.1",
+      "Descrizione":"SVGSanitizer: verifica che venga correttamente sanificato il file SVG di esempio",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-9.2",
+      "Descrizione":"SVGSanitizer: verifica che venga correttamente sanificato il file SVG fornito da Sanmarco Informatica",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-10.1",
+      "Descrizione":"getAllEmptyZones: verifica che vengano correttamente ritornate tutte le zone vuote",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-11.1",
+      "Descrizione":"getAllZones: verifica che vengano correttamente ritornate tutte le zone",
+      "Stato":"P"
+   },
+   {
+      "Codice":"INT-12.1",
+      "Descrizione":"getBinsByZoneId: verifica che vengano correttamente ritornati tutti i bin dato l'ID di una zona",
```

Comment:
Lo riscriverei così:

"verifica che, dato l'ID di una zona, vengano correttamente ritornati tutti i bin contenuti in essa"